.gitignore

```diff
@@ -5,3 +5,4 @@ site
 static/styles/vendor.css
 static/styles/app.css
 static/styles/fonts.css
+.idea
```

Comment:
We generally prefer not to include editor-specific files in the project .gitignore (I see .bk and .DS_Store here which seem like they could get removed by a separate PR, FWIW) -- can you drop this change?

Reply:
If you don't know it yet, you can have `$XDG_CONFIG_HOME/git/ignore` for global ignores and `.git/info/exclude` for not committed per-repo ignores.